framer/scripts/templates/selection_control.txt

```diff
@@ -1,18 +1,22 @@
 {{=« »=}}
 import * as React from 'react';
 import { addPropertyControls, ControlType } from 'framer';
-// tslint:disable-next-line: ban-ts-ignore
-// @ts-ignore
-import Mui«componentName» from '@material-ui/core/«componentName»';
-// tslint:disable-next-line: ban-ts-ignore
-// @ts-ignore
-import FormControlLabel from '@material-ui/core/FormControlLabel';
-
-export function «componentName»(props) {
-  const { checked: checkedProp, label, onChange, size, ...other } = props;
-  // tslint:disable-next-line: ban-ts-ignore
-  // @ts-ignore
+import Mui«componentName», { «componentName»Props as Mui«componentName»Props } from '@material-ui/core/«componentName»';
+import FormControlLabel, { FormControlLabelProps } from '@material-ui/core/FormControlLabel';
+
+interface Props {
+  checked: boolean;
+  color: FormControlLabelProps['color'];
+  disabled: boolean;
+  onChange?: React.ChangeEventHandler<HTMLInputElement>;
+  size: Mui«componentName»Props['size'];
+  label: string;
+  width: number | string;
+  height: number | string;
+}
```

Comment:
Was it intentional to define this statically?

Reply:
`string` would be sufficient here. Though I'm wondering if we shouldn't move all `width` and `height` typings to `number | string` since that is what you can actually do in `style`.

Reply:
Especially since the declared value (framerConfig) and type (template) live in two separate files

Reply:
I meant the fact that they're declared in the template,  rather than using `«& tsInterface»`, meaning they could go out of date WRT the component being wrapped.

Reply:
done